frappe/core/doctype/data_import/importer_new.py

```diff
@@ -363,13 +363,15 @@ def parse_value(self, value, df):
 
 	def parse_date_format(self, value, df):
```

Comment:
If the value is a `datetime` object, then it should not be passed to `parse_date_format`. Should be handled in `parse_value` itself.

Reply:
done